ng-dev/pr/common/labels/action.ts

```diff
@@ -0,0 +1,23 @@
+import {createTypedObject, Label} from './base.js';
+
+interface ActionLabel extends Label {}
+
+export const actionLabels = createTypedObject<ActionLabel>()({
+  ACTION_MERGE: {
+    description: 'The PR is ready for merge by the caretaker',
```

Comment:
```suggestion
    description: 'The PR author is ready for merge by the caretaker',
```

I wonder if this should be done that way. It's always unclear if people can apply it before when they are ready (as the author) etc. You know this 😄

Reply:
This was the text used in the description in the managed labels proposal.

Since we use the merge label currently to define the merge queue, I think we should leave it as is for now and if we find we need to update it in the future we can.